### DISCUSSION TITLE
How to change color with Fooocus?

### DISCUSSION THREAD

**Discussion by rabeyratne-zapmii** [lllyasviel/Fooocus#2256] [Q&A]
Hey guys, I've been enjoying using Fooocus quite a bit now. Is it possible to change just the color of something?
Example, this tooth is pink-ish, but I want to change it to white. I've tried in-paint, with Improve Detail and Modify Content and then prompted white! and change color to white! none of them seem to be working.

Is it possible to do this with fooocus itself?

Thanks in advance

![image](https://github.com/lllyasviel/Fooocus/assets/81731904/f87552ed-4632-4b03-a947-88bdd604afba)

**Comment by lgwjames** [comment]
impaint it, mask area and in left box put like white tooth?
